packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.ts

```diff
@@ -31,10 +110,31 @@ export function trackViewMetrics(
     viewStart,
     (newLoadingTime) => {
       viewMetrics.loadingTime = newLoadingTime
+
+      // We compute scroll metrics at loading time to ensure we have scroll data when loading the view initially
+      // This is to ensure that we have the depth data even if the user didn't scroll or if the view is not scrollable.
```

Comment:
It's not really a problem to not have scroll information for all the views, we can consider that this information is saved only when the user has spent enough time on the view (i.e. it has at least finished loading). Loading time is intersting because the page has usually reached a state where metrics like the scroll height are stable. It's true that it's not 100% reliable though, we sometimes send values before the page is fully loaded (See how small the scroll height can be [here](https://dd.datad0g.com/s/yB5yjZ/fvz-u29-phu))